tests/core/RequestHandler.spec.ts

```diff
@@ -130,6 +136,30 @@ describe('RequestHandler', () => {
     await batchScheduler.writeOperationBatch();
   });
 
+  it('should process create operation from test vectors correctly', async () => {
+    const createOperationBuffer = Buffer.from(JSON.stringify(createFixture));
+    const response = await requestHandler.handleOperationRequest(createOperationBuffer);
+    expect(response.status).toEqual(ResponseStatus.Succeeded);
+  });
+
+  it('should process update operation from test vectors correctly', async () => {
+    const updateOperationBuffer = Buffer.from(JSON.stringify(updateFixture));
+    const response = await requestHandler.handleOperationRequest(updateOperationBuffer);
+    expect(response.status).toEqual(ResponseStatus.Succeeded);
+  });
+
+  it('should process recover operation from test vectors correctly', async () => {
+    const recoverOperationBuffer = Buffer.from(JSON.stringify(recoverFixture));
+    const response = await requestHandler.handleOperationRequest(recoverOperationBuffer);
+    expect(response.status).toEqual(ResponseStatus.Succeeded);
```

Comment:
same comment as above, how do we know how the recover operation transforms the did document?